src/main/java/gift/exception/GlobalExceptionHandler.java

```diff
@@ -62,11 +63,16 @@ public ResponseEntity<ErrorResponseDto> handleUnauthorized(UnauthorizedException
 
     @ExceptionHandler(ForbiddenException.class)
     public ResponseEntity<ErrorResponseDto> handleForbidden(ForbiddenException ex) {
-        ErrorResponseDto responseDto = new ErrorResponseDto(
+        ErrorResponseDto responseDto = ErrorResponseDto.of(
                 HttpStatus.FORBIDDEN.value(),
                 "Forbidden",
                 ex.getMessage()
         );
         return ResponseEntity.status(HttpStatus.FORBIDDEN).body(responseDto);
     }
+
+    @ExceptionHandler(WishlistItemNotFoundException.class)
+    public ResponseEntity<Void> handleWishlistItemNotFound(WishlistItemNotFoundException ex) {
```

Comment:
```suggestion
    public ResponseEntity< ErrorResponseDto > handleWishlistItemNotFound(WishlistItemNotFoundException ex) {
```
통일성을 위해 이렇게 변경하면 좋을 것 같습니다~